package.json

```diff
@@ -98,7 +98,7 @@
     "mochawesome-report-generator": "^5.1.0"
   },
   "resolutions": {
-    "@contentful/forma-36-react-components": "3.47.1",
+    "@contentful/forma-36-react-components": "3.57.1",
```

Comment:
I think that the minimum version should be also bumped in the `packages/reference/package.json` too.